plugins/openstack_swift/openstack_swift_stats_

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+ME="${0##*/}"
+CONTAINER=$(printf "%s" "$ME" | cut -d'_' -f4 -)
+# printf "$CONTAINER\n"
+
+case $1 in
+   config)
+        cat <<EOM
+multigraph openstack_swift_stats_size
+graph_title Swift container size ($CONTAINER)
+graph_category cloud
+graph_vlabel bytes
+graph_args --base=1024 -l 0
+graph_period hour
+size.label Size
+size.type GAUGE
+size.draw AREA
+size.min 0
+growth.label Growth/\${graph_period}
+growth.type DERIVE
+growth.min 0
+
+multigraph openstack_swift_stats_objects
+graph_title Swift container object count ($CONTAINER)
+graph_category cloud
+graph_vlabel 
+graph_args --base=1000 -l 0
+graph_period hour
+objects.label Objects
+objects.type GAUGE
+objects.draw AREA
+objects.min 0
+objgrowth.label Growth/\${graph_period}
+objgrowth.type DERIVE
+objgrowth.min 0
+EOM
+        exit 0;;
+esac
+
+IFS=" "
```

Comment:
Is the `IFS` assignment really necessary here? It could be confusing for future readers, since it is effective until the end of the script.
(does it provide any required effect right now?)